pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -3523,7 +3523,7 @@ func (c *Cloud) EnsureLoadBalancer(ctx context.Context, clusterName string, apiS
 		return nil, fmt.Errorf("could not find any suitable subnets for creating the ELB")
 	}
 
-	loadBalancerName := cloudprovider.GetLoadBalancerName(apiService)
+	loadBalancerName := cloudprovider.DefaultGetLoadBalancerName(apiService)
```

Comment:
This should be `c.GetLoadBalancerName` (which calls cloudprovider.DefaultGetLoadBalancer)